uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -35,15 +35,16 @@ define([
       this.absenceTypes = [];
       this.calendar = {};
       this.contactName = null;
-      this.submitting = false;
       this.errors = [];
       this.managedContacts = [];
       this.requestDayTypes = [];
       this.selectedAbsenceType = {};
+      this.statusNames = sharedSettings.statusNames;
       this.period = {};
       this.postContactSelection = false; // flag to track if user is selected for enabling UI
       this.requestStatuses = {};
-      this.statusBeforeEdit = {};
+      this.managerDropDownStatus = null;
```

Comment:
Do not tie this directly to the manager. Call it `newStatusOnSave`. I'm not sure about this name either (it's not as expressive as i'd like), but the point is that the name you chose for the property does two things:

1. It ties it to the manager role. The admin can also change the status. Also, what if the staff will be given the ability to transition to some other status?
2. It ties to a specific element of the UI. What if you can change the status via something else than a dropdown?

Calling the property with a name that simply says what it is for, makes it more maintainable